src/artemis/ispyb/store_in_ispyb.py

```diff
@@ -0,0 +1,133 @@
+        params["axis_start"] = data_collection.axis_start
+        params["axis_end"] = data_collection.axis_end
+        params["axis_range"] = data_collection.axis_range
+        params["focal_spot_size_at_samplex"] = data_collection.focal_spot_size_at_sample_x
+        params["focal_spot_size_at_sampley"] = data_collection.focal_spot_size_at_sample_y
+        params["slitgap_vertical"] = data_collection.slitgap_vertical
+        params["slitgap_horizontal"] = data_collection.slitgap_horizontal
+        params["beamsize_at_samplex"] = data_collection.beamsize_at_sample_x
+        params["beamsize_at_sampley"] = data_collection.beamsize_at_sample_y
+        params["transmission"] = data_collection.transmission
+        params["comments"] = data_collection.comments
+        params["datacollection_number"] = data_collection.data_collection_number
+        params["detector_distance"] = data_collection.detector_distance
+        params["exp_time"] = data_collection.exposure_time
+        params["imgdir"] = data_collection.img_dir
+        params["imgprefix"] = data_collection.img_prefix
+        params["imgsuffix"] = data_collection.img_suffix
+        params["n_images"] = data_collection.number_of_images
+        params["n_passes"] = data_collection.number_of_passes
+        params["overlap"] = data_collection.overlap
+        params["flux"] = data_collection.flux
+        params["omegastart"] = data_collection.omega_start
+        params["start_image_number"] = data_collection.start_image_number
+        params["resolution"] = data_collection.resolution
+        params["wavelength"] = data_collection.wavelength
+        params["xbeam"] = data_collection.x_beam
+        params["ybeam"] = data_collection.y_beam
+        params["xtal_snapshot1"] = data_collection.xtal_snapshots_1
+        params["xtal_snapshot2"] = data_collection.xtal_snapshots_2
+        params["xtal_snapshot3"] = data_collection.xtal_snapshots_3
+        params["synchrotron_mode"] = data_collection.synchrotron_mode
+        params["undulator_gap1"] = data_collection.undulator_gap
+        params["starttime"] = data_collection.start_time
+        params["endtime"] = data_collection.end_time
+        params["run_status"] = data_collection.run_status
+        params["file_template"] = data_collection.file_template
+        params["binning"] = data_collection.binning
+
+        return mx_acquisition.upsert_data_collection(list(params.values()))
+
+    def store_position_table(self, position: Position) -> int:
+        mx_acquisition = self.conn.mx_acquisition
+
+        params = mx_acquisition.get_dc_position_params()
+        params["pos_x"] = position.position_x
+        params["pos_y"] = position.position_y
+        params["pos_z"] = position.position_z
+
+        return mx_acquisition.update_dc_position(list(params.values()))
+
+    def store_data_collection_group_table(self, data_collection_group: DataCollectionGroup) -> int:
```

Comment:
Can you prefix these functions with a `_` so that they're "private" as they will currently fail if called externally